corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -266,6 +267,7 @@ def add_post_to_entry(self, form, module, e):
                 data.ref = QuerySessionXPath(case_datum_id).instance()
                 data.exclude = CaseIDXPath(data.ref).case().count().neq(0)
                 e.post.data.append(data)
+            e.post.relevant = CaseClaimXpath.multi_select_relevant()
```

Comment:
maybe rename this to `mutli_case_relevant` since it now applies to more than just multi-select.

Reply:
good call [ba15267](https://github.com/dimagi/commcare-hq/pull/33738/commits/ba1526721640941be0ea8ad5d29ebb47ebf3a83e)